Make Graphviz use the mock module from the Python standard library rather than the standalone one

diff --git a/setup.cfg b/setup.cfg
index 3802a41..f3a59b2 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -13,7 +13,7 @@ testpaths = README.rst docs graphviz tests
 addopts =
   --doctest-modules --doctest-glob='*.rst' --ignore=docs/conf.py
   --cov --cov-report=term --cov-report=html
-mock_use_standalone_module = true
+mock_use_standalone_module = false
 log_cli = false
 log_cli_level = DEBUG
 log_file = test-log.txt
